### PR TITLE
Add InfEdit to image editing pipelines

### PR DESCRIPTION
**Pull request by h6kplus**
# What does this PR do?

Add the implementation of InfEdit (CVPR 2024, https://sled-group.github.io/InfEdit/)

## Modification

Add Infedit to infermodel._init_
create infedit.py in infermodel
create pipeline foler infedit in pipelines

### CODE REVIEW

File: src/imagen_hub/infermodels/infedit.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import PIL
+
+from diffusers import StableDiffusionPipeline, LCMScheduler
+
+class InfEdit():
+    """
+    A class for Prompt-to-Prompt.
+    
+    References: https://github.com/google/prompt-to-prompt/blob/main/null_text_w_ptp.ipynb
+    """
```
Comment (vinesmsuic) [inline src/imagen_hub/infermodels/infedit.py:11]: need update for docstring

**Review by vinesmsuic — CHANGES_REQUESTED** [review]
Hi, thank you for much for the contribution! Just need a minor modfication on docstring.

**Comment by h6kplus** [comment]
Updated!

**Review by vinesmsuic — APPROVED** [review]
LGTM, thanks for the contribution!

**Pull request merged**